app/src/main/res/layout/promoted_story_card.xml

```diff
@@ -80,14 +80,14 @@
           android:maxLines="1"
           android:text="@{viewModel.promotedStory.storyName}"
           android:textColor="@color/oppiaPrimaryText"
-          android:textSize="16sp" />
+          android:textSize="@dimen/promoted_story_card_text_size"/>
```

Comment:
Nit space missing at the end after "
`android:textSize="@dimen/promoted_story_card_text_size" />`

Reply:
Done